src/App.tsx

```diff
@@ -34,6 +35,7 @@ export const app = initializeApp({
   apiKey: "AIzaSyCsukUZtMkI5FD_etGfefO4Sr7fHkZM7Rg",
   authDomain: "auth.hexlabs.org",
 });
+export const HEXATHON_ID = "62d9ed68d0a69b88c06bdfb2";
```

Comment:
can you change this to use the `String(process.env.REACT_APP_HEXATHON_ID)` everywhere u need the hexathonID